.github/workflows/ci.yml

```diff
@@ -122,48 +150,26 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: Check doxygen build
-        uses: FreeRTOS/CI-CD-Github-Actions/doxygen@main
+        uses: FreeRTOS/CI-CD-Github-Actions/doxygen@v2
         with:
           path: ./
 
   spell-check:
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout Parent Repo
-        uses: actions/checkout@v3
-        with:
-          ref: main
-          repository: aws/aws-iot-device-sdk-embedded-C
-      - run: rm -r libraries/standard/corePKCS11
       - name: Clone This Repo
         uses: actions/checkout@v3
+      - name: Run spellings check
+        uses: FreeRTOS/CI-CD-GitHub-Actions/spellings@v2
         with:
-          path: libraries/standard/corePKCS11
-      - name: Install spell
-        run: |
-          sudo apt-get install spell
-          sudo apt-get install util-linux
-      - name: Check spelling
-        run: |
-          PATH=$PATH:$PWD/tools/spell
-          # Modifies `find` command used in spell checker to ignore the test and dependency directory
-          # The command looks like this `extract-comments `find $DIRNAME -name \*.[ch]` should the line change and the sed command will
-          # append "-not path {val added below}" for each of the directories mentioned.
-          # https://github.com/aws/aws-iot-device-sdk-embedded-C/blob/ad28ed355df4f82b77f48028e24bd6fc9e63bc54/tools/spell/find-unknown-comment-words#L86
-          sed -i 's/find $DIRNAME/find $DIRNAME -not -path '*test*' -not -path '*dependency*'/g' tools/spell/find-unknown-comment-words
-          find-unknown-comment-words --directory libraries/standard/corePKCS11
-          if [ "$?" = "0" ]; then
-            exit 0
-          else
-            exit 1
-          fi
+          path: ./
 
   formatting:
     runs-on: ubuntu-20.04
```

Comment:
Any reason for using ubuntu-20.04 here while all others are using ubuntu-latest?

Reply:
The formatting check uses uncrustify version .69f, which isn't available on ubuntu-22.04/ubuntu-latest